src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStoreV36.cs

```diff
@@ -0,0 +1,82 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using FellowOakDicom;
+using Microsoft.Data.SqlClient;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.Core.Features.Model;
+using Microsoft.Health.Dicom.Core.Models;
+using Microsoft.Health.Dicom.SqlServer.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer.Features.Storage;
+
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.Store;
+
+internal class SqlIndexDataStoreV36 : SqlIndexDataStoreV23
```

Comment:
This should be SqlIndexDataStoreV35 right?

Reply:
Yes, you normally inherit from previous derived class. If you do this way, it will have an issue if someone inherits from V36 but the latest delete is not available